tests/unit/test_policy_drivers.py

```diff
@@ -517,10 +565,24 @@ def test_get_policy_driver_value_error(self, m_driver):
         cni_plugin = Mock(spec=CniPlugin)
         cni_plugin.network_config = {"name": "testnetwork"}
         cni_plugin.running_under_k8s = False
+        cni_plugin.running_under_mesos = False
 
         # Call
         with assert_raises(SystemExit) as err:
             get_policy_driver(cni_plugin)
         e = err.exception
         assert_equal(e.code, ERR_CODE_GENERIC)
 
+    def test_get_policy_driver_mesos(self):
+        cni_plugin = Mock(spec=CniPlugin)
+        cni_plugin.network_config = {"name": "testnetwork",
```

Comment:
I think ideally we'd want this to be an error case (since two policy types are specified).

I don't think we should write that code right now though...